### ISSUE TITLE
Can't add some system fonts

### ISSUE THREAD

**Issue by MoneyAllDay** [WilliamRagstad/Font-Manager#17]
I tried to add a system font and i got the message "[The filepath to 'Bahnshrift SemiLight' could not be found. Please select another font.](https://i.imgur.com/CTKUiPF.gif)"

A font which i have [installed](https://i.imgur.com/trYfwox.png).

**Comment by IJuanTM** [comment]
Yup have this problem as well with all of the custom font I installed to my windows...

**Comment by BugFlxer** [comment]
Hello! I Have Experienced This Too.. Here Is A Short Fix:
Download The Font Again After That Download The Font Manager Version: Added Auto Updater


 and the latest Version:

**Comment by ikonntech** [comment]
> Hello! I Have Experienced This Too.. Here Is A Short Fix: Download The Font Again After That Download The _Font Manager Latest Release_ This Issue is Commonly Created Because of the path connection. After That Click Reset on The Right Corner of the Font manager And Launch It Again.. This Worked For me im sorry if it didnt for you

It's not worked for me. The program does not work.
